### ISSUE TITLE
stopsNearby raises KeyError: 'StopLocation' on certain GPS coordinates

### ISSUE THREAD

**Issue by PeterSR** [tomatpasser/python-rejseplanen#1]
Certain GPS coordinates (probably those far away from Denmark) causes `stopsNearby` to raise KeyError: 'StopLocation' instead of returning an empty list or raising a rjpl-specific exception. I would suggest returning an empty list to be consistent with the expected return value.

Minimum reproducible example:

```python
import rjpl

print(rjpl.stopsNearby(0.0, 0.0))
```

If one uses `_request` directly, it appears that the returned JSON object looks like this

```
{'LocationList': {'$': '\n',
                  'noNamespaceSchemaLocation': 'http://xmlopen.rejseplanen.dk/xml/rest/hafasRestStopsNearby.xsd'}}
```

I can create a pull request for a suggested fix if you want.

**Comment by thomaspasser** [comment]
Fixed in 0b34f76, will be in v0.3.6

**Comment by thomaspasser** [comment]
Thank you for submitting the issue 👍
